agent/cache/cache_test.go

```diff
@@ -934,15 +934,15 @@ func TestCacheGet_refreshAge(t *testing.T) {
 		Return(func(o FetchOptions, r Request) FetchResult {
 			idx := atomic.LoadUint64(&index)
 			if atomic.LoadUint64(&shouldFail) == 1 {
-				t.Logf("Failing Fetch at index %d", idx)
+				// t.Logf("Failing Fetch at index %d", idx)
```

Comment:
I'll fix this in another PR. This is a separate goroutine calling `t.Logf` and apparently it was outliving the test, causing a panic when it tried to write to the test logger for a test that was done.